sdk/eventgrid/azure-eventgrid/swagger/_event_map.py

```diff
@@ -0,0 +1,164 @@
+    AcsChatMemberAddedToThreadWithUserEventName = (
```

Comment:
Why there is a swagger folder?

Reply:
just to add this sdk/eventgrid/azure-eventgrid/swagger/postprocess_eventnames.py file - we have an issue to hook it into the swagger readme itself -but for now, it exists here

also, removed the _eventmap.py file